### PR TITLE
[HMC-113] - Slicing course page

### PR DESCRIPTION
**Pull request by mpsalunggg**
## Description :

- Slicing course page
- Create component global label path
- Responsive course page

## Screenshots / Video Record:

https://github.com/hammer-code/lms-fe/assets/92923869/4e4b751b-3b79-4d48-b95c-7dec093d2fb1

### CODE REVIEW

File: src/features/Dashboard/Course/configs.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
+// Data Temporary, waiting for backend
+export const DATA_COURSE = [
```
Comment (vickyadrii) [inline src/features/Dashboard/Course/configs.ts]: ini mock data kan? pindahkan ke folder constants saja keknyaa spaya lebih cakep + define tipe datanya

Comment (mpsalunggg) [reply src/features/Dashboard/Course/configs.ts]: ohh iyo lupa awkwkw, baik kak

Comment (mpsalunggg) [reply src/features/Dashboard/Course/configs.ts]: @vickyadrii done kaks

Comment (vickyadrii) [reply src/features/Dashboard/Course/configs.ts]: sikatt @mpsalunggg

**Review by vickyadrii — APPROVED** [review]

**Pull request merged**